include/overlay005/encounter_effect_core.h

```diff
@@ -0,0 +1,38 @@
+#ifndef POKEPLATINUM_ENCOUNTER_EFFECT_CORE_H
+#define POKEPLATINUM_ENCOUNTER_EFFECT_CORE_H
+
+#include "struct_decls/sys_task.h"
+
+void EncounterEffect_TallGrass_HigherLevel(SysTask *task, void *param);
+void EncounterEffect_TallGrass_LowerLevel(SysTask *task, void *param);
```

Comment:
I think this can just be `Grass` to get the idea across well-enough (and as well for the Trainer encounters).